src/app/src/test/kotlin/org/imperial/mrc/hint/unit/controllers/MetadataControllerTests.kt

```diff
@@ -24,4 +24,19 @@ class MetadataControllerTests
         val result = sut.plotting("MWI")
         assertThat(result).isSameAs(mockResponse)
     }
+
+    @Test
+    fun `get hintr version`()
+    {
+        val mockResponse = mock<ResponseEntity<String>>()
+
+        val mockAPIClient = mock<HintrAPIClient>
+        {
+            on {getVersion()} doReturn mockResponse
```

Comment:
Nitpick - the spacing between braces isn't quite right here (and in the controller), it's good to get in the habit of auto-formatting your code as you go (<kbd>ctrl + alt + l</kbd> in IntelliJ on Linux, possibly <kbd>cmd + alt + l</kbd> on Mac?)